### ISSUE TITLE
Name and title are not defined for layout controls present in ‘New section’ tab.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#106]
User Experience:

User would find difficulty to identify the controls if name and title are not defined for any control as they won't get the context of that control and also they won't be able to perform functionality  relevant to that control. 

Note: User credentials should NOT be included in the bug.

 

Repro-steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode.

4. Navigate to the Left nav and select Pages button.

5. Navigate to the Pages window. Select "APIs: List" option.

6. Navigate to the any editable control details and select Add section.

7. Select various type of section in the dialog for layout.

8. Open developer tool using F12 key and navigate to layout controls present in ‘New section’ tab.

9. Verify the issue.

 

Actual Result:

Name and title are not defined for layout controls present in ‘New section’ tab, screen reader announces only position of controls.

 

Expected Result:

Name and title should be defined for layout controls present in ‘New section’ tab, screen reader should announces control name with position.

 

Note:

Similar issue exist with Chrome.

 

MAS Reference:

MAS 4.1.2 - Name, Role, Value

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
